src/components/ui/formik/TextAreaSecretField.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { FormGroup, Button } from '@patternfly/react-core';
+import { TextAreaSecretProps } from './types';
+import { getFieldId } from './utils';
+import TextAreaField from './TextAreaField';
+
+const TextAreaSecretField: React.FC<TextAreaSecretProps> = ({
+  isSet,
+  helperTextHidden,
+  onReset,
+  ...props
+}) => {
+  const { label, name } = props;
+  const [isUpdate, setUpdate] = React.useState(false);
+  const fieldId = getFieldId(name, 'input');
+  const handleClick = React.useCallback(() => {
+    setUpdate(!isUpdate);
+    onReset && onReset();
+  }, [setUpdate, isUpdate, onReset]);
+
+  if (isSet && !isUpdate) {
+    return (
+      <FormGroup fieldId={fieldId} label={label} helperText={helperTextHidden}>
```

Comment:
```suggestion
      <FormGroup fieldId={fieldId} label={label} helperText={helperTextHidden} isRequired={props.isRequired}>
```

Reply:
This has probably no practical effect, but ok, added